netmiko/yamaha/yamaha.py

```diff
@@ -28,7 +28,7 @@ def exit_enable_mode(self, exit_command="exit"):
             time.sleep(1)
             output = self.read_channel()
             if "(Y/N)" in output:
-                self.write_channel("N")
+                self.write_channel(f"N{self.default_enter}")
```

Comment:
@Tachashi I moved the sending of the newline (\n) to here as I assumed the issue was that sending of the "N" required an <enter> at the end of it.

Let me know if this doesn't work for some reason i.e. you are still seeing issues with `exit_enable_mode`